### PR TITLE
Lints found by shelly.

### PR DESCRIPTION
**Pull request by m-kostrzewa**
Shelly built from commit 26c825c942d6e99ba94a01080cc43d09baae16aa. It added support for detecting unused imports.

This commit consists mostly of removing unused imports but also some older types lint fixes.

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
Some small issues found on the way: 

* Build.ps1 - use dict instead of array
* Refactor -InstallNodeMgr flag
* NodeMgr tests - refactor hardcoded LogPath
* Diagnostic check - enable strict mode
* Backups - enable strict mode
* Configuration.ps1 - management interface is hardcoded to Ethernet0

File: Test/Utils/ComputeNode/Configuration.ps1 [hunk 1]
```diff
@@ -1,6 +1,12 @@
 . $PSScriptRoot\..\..\..\CIScripts\Common\Init.ps1
 . $PSScriptRoot\..\..\..\CIScripts\Common\Invoke-CommandWithFunctions.ps1
-. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testenv.ps1
+
+# [Shelly-Bug] Shelly doesn't handle imports of classes yet.
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testenv.ps1 # allow unused-imports
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testbed.ps1
+
+# . $PSScriptRoot\..\..\TestConfigurationUtils.ps1
```
Comment (michal-clapinski) [inline Test/Utils/ComputeNode/Configuration.ps1]: what is that?

File: Test/PesterLogger/PesterLogger.ps1 [hunk 2]
```diff
@@ -1,6 +1,7 @@
 . $PSScriptRoot\..\..\CIScripts\Common\Aliases.ps1
 
-. $PSScriptRoot\Get-CurrentPesterScope.ps1
+# [Shelly-Bug] analyzer doesn't detect allow unused-imports
```
Comment (michal-clapinski) [inline Test/PesterLogger/PesterLogger.ps1]: are you sure this is the correct comment here?

File: CIScripts/Build.ps1 [hunk 3]
```diff
@@ -41,48 +40,39 @@ foreach ($Directory in $Directories) {
     }
 }
 
-$ComponentsToBuild = Get-ComponentsToBuild
-
 try {
-    if ("DockerDriver" -In $ComponentsToBuild) {
```
Comment (michal-clapinski) [inline CIScripts/Build.ps1]: If you're removing ComponentsToBuild, please also remove them in `BuildStage.ps1`

**Comment by m-kostrzewa** [comment]
recheck no bug

**Pull request merged**